static/js/apps/build/components/build_hero.tsx

```diff
@@ -0,0 +1,65 @@
+/**
+ * Copyright 2024 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * A component to display the build page hero.
+ */
+
+import React, { ReactElement } from "react";
+
+import { HeroColumns } from "../../../components/content/hero_columns";
+
+export const BuildHero = (): ReactElement => {
+  return (
+    <HeroColumns>
+      <HeroColumns.Left>
+        <h2>
+          Build your Data Commons, overlay your data with global data, and let
+          everyone in your organization uncover insights with natural language
+          questions.{" "}
+          <a
+            href="https://docs.datacommons.org/custom_dc?utm_source=buildpage_hero"
+            title="Build your own Data Commons"
+          >
+            Learn how
+          </a>
+        </h2>
+      </HeroColumns.Left>
+      <HeroColumns.Right>
+        <div>
+          <h4>Build and deploy your own</h4>
```

Comment:
Looks like the font sizes are slightly different here compared to what's in production:

<img width="1728" alt="Screenshot 2024-12-06 at 10 28 57 PM" src="https://github.com/user-attachments/assets/805d6554-f565-4969-b5fa-08c8e7cd44af">